cdap-common/src/main/java/co/cask/cdap/common/lang/ExtensionClassLoader.java

```diff
@@ -30,17 +26,16 @@
 import java.util.Set;
 
 /**
- * {@link DirectoryClassLoader} for {@link Authorizer} extensions.
+ * {@link DirectoryClassLoader} for extensions.
  */
-public class AuthorizerClassLoader extends DirectoryClassLoader {
+public class ExtensionClassLoader extends DirectoryClassLoader {
```

Comment:
This change is a bit scary. Can you explain the classloading model for the log saver plugin? Do we have a design page?